cloud/vmware/vmware_local_user_manager.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright IBM Corp. 2016
+# Author(s): Andreas Nafpliotis <nafpliot@de.ibm.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/
+
+DOCUMENTATION = '''
+---
+module: vmware_local_user_manager
+short_description: Manage local users on an ESXi host
+description:
+    - Manage local users on an ESXi host
+version_added: "2.2"
+author: Andreas Nafpliotis
+notes:
+    - Tested on ESXi 6.0
+    - Be sure that the ESXi user used for login, has the appropriate rights to create / delete / edit users
+requirements:
+    - "python >= 2.6"
+    - PyVmomi installed
+options:
+    hostname:
+        description:
+            - The hostname or IP address of the ESXi server
+        required: True
+    username:
+        description:
+            - The username of the ESXi server
+        required: True
+        aliases: ['user', 'admin']
+    password:
+        description:
+            - The password of the vSphere vCenter
+        required: True
+        aliases: ['pass', 'pwd']
+    local_user_name:
+        description:
+            - The local user name to be changed
+        required: True
+    local_user_password:
+        description:
+            - The password to be set
+        required: False
+    local_user_description:
+        description:
+            - Description for the user
+        required: False
+    state:
+        description:
+            - If the user should be present or absent. If the user already exists, with state present we update the user info
```

Comment:
`when C(state=present) we update...`

So it will be formatted like in http://docs.ansible.com/ansible/proxmox_template_module.html